plugins/intel-usb4/intel-usb4.quirk

```diff
@@ -1,2 +1,5 @@
 [USB\VID_8087&PID_0B40]
 Plugin = intel_usb4
+
+[USB\VID_8087&PID_1234]
```

Comment:
really `1234`?!

Reply:
this is a super early proto with invalid id on the board, will fix this commit when going stable.